projects/aca-content/src/lib/dialogs/folder-details/folder-information.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="aca-folder-info-container">
```

Comment:
No need for extra div which wraps whole html. You can apply styles directly on component's selector and delete that div

Reply:
Removed extra div